packages/shared/src/components/tooltips/Tooltip.tsx

```diff
@@ -0,0 +1,43 @@
+import React, { ReactElement, useMemo } from 'react';
+import dynamicParent from '../../lib/dynamicParent';
+import { getShouldLoadTooltip, LazyTooltipProps } from './LazyTooltip';
+
+const LazyTooltipLoader = () =>
+  import(/* webpackChunkName: "lazyTooltip" */ './LazyTooltip');
+
+const TippyTooltip = dynamicParent<
+  Omit<LazyTooltipProps, 'render' | 'allowHTML'>
+>(() => LazyTooltipLoader().then((mod) => mod.default), React.Fragment);
+
+interface TooltipProps extends Omit<LazyTooltipProps, 'render'> {
+  disableTooltip?: boolean;
+}
+
+export function Tooltip({
+  children,
+  content,
+  allowHTML,
+  disableTooltip,
+  ...props
+}: TooltipProps): ReactElement {
+  const component = useMemo(
+    () =>
+      React.cloneElement(children, {
+        ...children.props,
+        'aria-label': allowHTML ? children.props['aria-label'] : content,
+      }),
+    [children],
+  );
+
+  return (
+    <TippyTooltip
+      shouldLoad={getShouldLoadTooltip(disableTooltip)}
```

Comment:
Didn't we want to load the tooltip on first hover?
Here it looks like that on desktop devices (or any other non touch) it will load immediately

Reply:
Yes, back then, we were loading it on hover but we realized the web limitation for `children` having image in it, so it flicks. This is why we had decided to move forward loading it immediately as it left us an option to build a tooltip from scratch which would require tons of work.